app/addons/config/routes.js

```diff
@@ -67,7 +67,7 @@ function (app, FauxtonAPI, Config, Views, CORSComponents, CORSActions, ClusterAc
       this.sidebar = this.setView('#sidebar-content', new Views.Tabs({
         sidebarItems: [
           {
-            title: 'Main config',
+            title: 'config.js File',
```

Comment:
I think instead of `Main config` maybe something like General. Its actually all the config but CORS.